pages/common/git-squash.md

```diff
@@ -0,0 +1,21 @@
+# git squash
+
+> Squash N last changes up to a ref'ed commit into current HEAD.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-squash>.
+
+- Squash everything from a specific branch into the current branch:
+
+`git squash {{source-branch}}`
+
+- Squash everything from a commit reference into the current branch:
+
+`git squash {{commit-ref}}`
+
+- Squash the `n` latest commits and commit with a message:
+
+`git squash HEAD~{{n}} "{{message}}"`
+
+- Squash the `n` latest commits and commit concatenating all messages:
+
+`git squash --squash-msg @~{{n}}`
```

Comment:
Although `@` means `HEAD`, should we be consistent and use `HEAD` like we have used in the above examples ? (thoughts)
```suggestion
`git squash --squash-msg HEAD~{{n}}`
```

Reply:
That indeed can be good for consistency. I'll apply this change.